controllers/controllers.js

```diff
@@ -1,9 +1,33 @@
-const {fetchCategories}=require('../models/models')
+const { commentData } = require("../db/data/test-data");
+const comments = require("../db/data/test-data/comments");
+const { fetchCategories, fetchReviews,fetchComments } = require("../models/models");
 
-exports.getCategories=(request,response,next)=>{
-    fetchCategories().then((categories)=>{
-        response.status(200).send({categories})
-    }).catch((error)=>{
-        next(error);
-    })
-}
\ No newline at end of file
+exports.getCategories = (request, response, next) => {
+	fetchCategories()
+		.then((categories) => {
+			response.status(200).send({ categories });
+		})
+		.catch((error) => {
+			next(error);
+		});
+};
+exports.getReviews = (request, response, next) => {
```

Comment:
A good way you can refactor this is to create a query string (dependant on certain conditions) then you can do db.query on that query string you have created, rather than doing 3 db.queries and everything getting nested. This is optional, because if it works, this is fine but it would make it easier to read and more dry.